python/ray/worker.py

```diff
@@ -513,14 +494,13 @@ def _deserialize_object_from_arrow(self, data, metadata, object_id,
             # If data is not empty, deserialize the object.
             # Note, the lock is needed because `serialization_context` isn't
             # thread-safe.
-            with self.plasma_client.lock:
-                return pyarrow.deserialize(data, serialization_context)
+            return pyarrow.deserialize(data, serialization_context)
```

Comment:
Where did the lock go? The above comment still refers to it.

Reply:
@robertnishihara indicated that this lock is no longer necessary. Can you confirm that? If it isn't, I'll remove the comment. If it is, I'll replace it with a new lock.